packaged/bin/bxiconfig

```diff
@@ -1,14 +1,14 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Did you try under python3?
is it possible to correct the behaviour if it's not working?